src/collective/liveblog/tests/test_upgrades.py

```diff
@@ -41,3 +42,35 @@ def test_profile_version(self):
     def test_registered_steps(self):
         steps = len(self.setup.listUpgrades(self.profile_id)[0])
         self.assertEqual(steps, 1)
+
+
+class To1002TestCase(UpgradeBaseTestCase):
+
+    from_ = '1001'
+    to_ = '1002'
+
+    def test_profile_version(self):
+        version = self.setup.getLastVersionForProfile(self.profile_id)[0]
+        self.assertEqual(version, self.from_)
+
+    def test_registered_steps(self):
+        steps = len(self.setup.listUpgrades(self.profile_id)[0])
+        self.assertEqual(steps, 1)
+
+    def test_remove_workflow(self):
+        title = u'Migrate liveblog workflow'
+        step = self._get_upgrade_step_by_title(title)
+        assert step is not None
+
+        # simulate (partially) state on previous version
+        wtool = api.portal.get_tool('portal_workflow')
+        wtool.setChainForPortalTypes(('Liveblog',), ('liveblog_workflow',))
+        assert wtool.getChainForPortalType('Liveblog') == ('liveblog_workflow',)
```

Comment:
because that is not a test, but a precondition for a test; the result is the same, is just semantics.